packages/web3-eth-ens/src/ens.ts

```diff
@@ -289,7 +292,10 @@ export class ENS extends Web3Context<EthExecutionAPI & Web3NetAPI> {
 		if (this._detectedAddress) {
 			return this._detectedAddress;
 		}
-		const networkType = await getId(this, DEFAULT_RETURN_FORMAT); // get the network from provider
+		const networkType = await getId(this, {
+			...DEFAULT_RETURN_FORMAT,
+			number: FMT_NUMBER.HEX,
```

Comment:
If we always want to have `number:FMT_NUMBER.HEX` here why we are using default format?